runtime/vm/guardedstorage.c

```diff
@@ -48,6 +49,15 @@ j9gs_initializeThread(J9VMThread *vmThread)
 	if (IS_THREAD_GS_INITIALIZED(vmThread)) {
 		success = 1;
 	} else {
+
+		int32_t supportsGuardedStorageFacility = 0;
+		J9ProcessorDesc  processorDesc;
+		j9sysinfo_get_processor_description(&processorDesc);
+		if (j9sysinfo_processor_has_feature(&processorDesc, J9PORT_S390_FEATURE_GUARDED_STORAGE) && j9sysinfo_processor_has_feature(&processorDesc, J9PORT_S390_FEATURE_SIDE_EFFECT_ACCESS)) {
```

Comment:
This line is too long. Would you please two lines for this if condition?

Reply:
fixed